docs/manage/fleet/_index.md

```diff
@@ -101,12 +101,12 @@ You can also control a robot using the [Viam mobile app](#the-viam-mobile-app).
 
 ### Control with the SDKs
 
-Everyone who has access to the robot's location can obtain the robot's remote address and location secret from the app's **Code sample** tab, which are both needed to send API calls to the robot from the [Viam SDKs](/program/apis/).
-You can share the robot's remote address and location secret without granting location access in the app.
+Everyone who has access to the robot's location can obtain the robot's remote address and API key from the app's **Code sample** tab, which are both needed to send API calls to the robot from the [Viam SDKs](/program/apis/).
+You can share the robot's remote address and API key without granting location access in the app.
 
 As long as each collaborator has access to these tokens for a robot, members of your team can write code, use tools like GitHub, and execute code to control the robot from anywhere in the world.
 
-Toggle **Include secret** above the code on the **Code Sample** tab of your robot's page to display or hide the robot's location secret.
+Toggle **Include secret** above the code on the **Code Sample** tab of your robot's page to display or hide the robot's API key.
```

Comment:
[minor] should say "Include API Key"?

Reply:
It should - this changed in the app after this was written ;)
```suggestion
Toggle **Include API Key** above the code on the **Code Sample** tab of your robot's page to display or hide the robot's API key.
```